### PR TITLE
Wildcard V2: <Panel /> Manual migration

### PR DESCRIPTION
**Pull request by gitstart-sourcegraph**
## Manual migration required to update our code to use the `<Panel />` Wildcard component.
See [Wildcard V2 - Planned work](https://docs.google.com/document/d/1NisbJPiadtt5jQw4vUUYJOr8dD6Urwn5V0mVq2wt6bE/edit#heading=h.g4cw92w3ouhw) for more context.

Note: Before doing this, we should check if there is scope for a codemod to automate this migration. If so, the codemod should be implemented and ran before starting work on this issue.

## Refs
[Sourcegraph Issue](https://github.com/sourcegraph/sourcegraph/issues/27710)
[Gitstart Ticket](https://app.gitstart.com/clients/sourcegraph/tickets/SG-27710)

## Preview App
[Storybook](https://client-sourcegraph-storybook-pr-287.onrender.com/)
[Frontend](https://client-sourcegraph-frontend-pr-287.onrender.com/)

### Implementation
1. Update all [usage](https://sourcegraph.com/search?q=context:global+repo:github.com/sourcegraph/sourcegraph+%3CResizable&patternType=literal) of the `<Resizable />` component to use `<Panel />`
2. Delete the `<Resizable />` component when no longer used.
3. Rename these [files and components](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@vk/add-wildcard-popover-tether-comp/-/blob/client/branded/src/components/panel/Panel.tsx?L119#L341:37) from `Panel `to `TabbedPanel` to avoid future confusion

### Acceptance criteria.

- [x] All incorrect usage of the previous pattern has been updated to use the Wildcard component

## Test plan
- Test on local running app and PR preview app to make sure all functions work well
- Confirm there are no unexpected UI variations.

### CODE REVIEW

**Comment by gitstart-sourcegraph** [comment]
Hi @valerybugakov,
From description `Rename these files and components from Panel to TabbedPanel to avoid future confusion`,
The Assumption is that this also includes the directory name `branded/src/components/panel` => `branded/src/components/TabbedPanel`. kindly help validate

**Comment by umpox** [comment]
@gitstart-sourcegraph Sorry this PR has been left around for a while! Let's try to get this merged, the code looks in a good state but there's now quite a few merge conflicts. Can you update with `main` and I'll do a full review?

**Comment by gitstart-sourcegraph** [comment]
@umpox Thanks. It took some times to resolve too much conflicts. I updated PR with PR view links. Could you please review?

File: client/web/src/repo/RepoRevisionSidebar.tsx [hunk 1]
```diff
@@ -86,78 +86,79 @@ export const RepoRevisionSidebar: React.FunctionComponent<Props> = props => {
...
-                                        onHandleSymbolClick={handleSymbolClick}
-                                    />
-                                </TabPanel>
-                            </TabPanels>
-                        </div>
-                    </Tabs>
-                </div>
-            }
-        />
+                        <Tab data-tab-content="files">
```
Comment (umpox) [inline client/web/src/repo/RepoRevisionSidebar.tsx:119]: Minor, I noticed that the focus ring now clips slightly:

image

Seems to be related to some style changes here

File: client/web/src/repo/RepoRevisionSidebar.module.scss [hunk 2]
```diff
@@ -24,3 +24,7 @@
         background-color: var(--color-bg-3);
     }
 }
+
+.panel {
+    --panel-bg-color: transparent;
+}
```
Comment (umpox) [inline client/web/src/repo/RepoRevisionSidebar.module.scss:30]: Is this necessary here? It seems like the repo revision sidebar shouldn't have a transparent background? Unless I'm missing something!

File: client/wildcard/src/components/Panel/Panel.module.scss [hunk 3]
```diff
@@ -1,37 +1,50 @@
...
     }
 }
 
 .handle {
-    background-color: var(--panel-handle-bg-color);
     position: absolute;
+    opacity: 0;
     user-select: none;
     z-index: 1;
+    transition: opacity 150ms;
```
Comment (umpox) [inline client/wildcard/src/components/Panel/Panel.module.scss:46]:
```suggestion
```

We're trying to move away from transitions, we've found it makes our UI seem snappier without them. I think we can just remove this line

**Review by umpox — COMMENTED** [review]
Some small comments, rest looks great!

**Comment by umpox** [comment]
cc: @mrnugget I've checked and I don't think this should break anything with your recent panel changes, but worth running locally and checking that the new panel looks OK if you get chance.

**Comment by mrnugget** [comment]
> cc: @mrnugget I've checked and I don't think this should break anything with your recent panel changes, but worth running locally and checking that the new panel looks OK if you get chance.

Tested locally, everything looks good! Thanks for the heads up

**Review by umpox — APPROVED** [review]
Thank you for quickly updating this PR!

File: client/web/src/repo/RepoRevisionSidebar.tsx (hunk 1 above)
Comment (gitstart-sourcegraph) [reply client/web/src/repo/RepoRevisionSidebar.tsx:113]: ops! Missed this. Checking...

**Pull request merged**